.github/main.workflow

```diff
@@ -4,13 +4,18 @@ workflow "Verify the module" {
 }
 
 action "install" {
-  uses = "actions/npm@1.0.0"
```

Comment:
please keep this pinned to 1.0.0 since master won't necessarily be stable

Reply:
@geekflyer Got it. Working on fixes right now.